_episodes/13-scheduler.md

```diff
@@ -37,63 +35,6 @@ why sometimes your job do not start instantly as in your laptop.
    file="/fig/restaurant_queue_manager.svg"
    alt="Compare a job scheduler to a waiter in a restaurant" %}
 
-> ## Job Scheduling Roleplay (Optional)
->
-> Your instructor will divide you into groups taking on different roles in the
-> cluster (users, compute nodes and the scheduler). Follow their instructions
-> as they lead you through this exercise. You will be emulating how a job
-> scheduling system works on the cluster.
->
-> > ## Instructions
-> >
-> > To do this exercise, you will need about 50-100 pieces of paper or sticky
-> > notes.
-> >
-> > 1. Divide the room into groups, with specific roles.
-> >    * Pick three or four people to be the "scheduler."
-> >    * Select one-third of the room be "users", given several slips of paper
-> >      (or post-it notes) and pens.
-> >    * Have the remaining two thirds of the room be "compute nodes."
-> >    * Have the "users" go to the front of the room (or the back, wherever
-> >      there's space for them to stand) and the "schedulers" stand between
-> >      the users and "compute nodes" (who should remain at their seats).
-> >
-> > 1. Divide the pieces of paper / sticky notes among the "users" and have
-> >    them fill out all the pages with simple math problems and their name.
-> >    Tell everyone that these are the jobs that need to be done and
-> >    correspond to their computing research problems.
-> >
-> > 1. Point out that we now have jobs and we have "compute nodes" (the people
-> >    still sitting down) that can solve these problems. How are the jobs
-> >    going to get to the nodes? The answer is the scheduling program that
-> >    will take the jobs from the users and deliver them to open compute
-> >    nodes.
-> >
-> > 1. Have all the "compute nodes" raise their hands. Have the users "submit"
-> >    their jobs by handing them to the schedulers. Schedulers should then
-> >    deliver them to "open" (hands-raised) compute nodes and collect finished
-> >    problems and return them to the appropriate user.
-> >
-> > 1. Wait until most of the problems are done and then re-seat everyone.
-> >
-> > > ## Discussion
-> > >
-> > > A "node" might be unable to solve the assigned problem for a variety of
-> > > reasons.
-> > >
-> > > * Ran out of time.
-> > > * Ran out of memory.
-> > > * Ran out of storage space, or could not load an input file or dataset.
-> > > * Doesn't know where to start: nobody "taught" it, i.e., the program
-> > >   can't be loaded.
-> > > * Gets stuck on a hard part: the program has the wrong algorithm, or was
-> > >   never told to load the library containing the right algorithm.
-> > > * Was busy thinking about something else, and didn't get to the problem
-> > >   yet.
-> > {: .discussion}
-> {: .challenge}
-{: .callout}
-
 The scheduler used in this lesson is {{ site.sched.name }}. Although
```

Comment:
Perhaps here, or perhaps in the paragraph above, emphasise *scheduler* as new vocab as with other key words in the follow section.